makefile

```diff
@@ -5,7 +5,7 @@ PWD := $(shell pwd)
 # Using 'where' on Windows and 'which' on Unix-like systems, respectively
 # We do not check for 'date', since it's a cmdlet on Windows, which do not show up with where
 # (:= assignment is necessary to force immediate evaluation of expression)
-EXECUTABLES = git docker kubectl
```

Comment:
I think we want to keep kubectl in our makefile.  It is used in this file for non client work so we should still require it.